homeassistant/components/aprilaire/strings.json

```
@@ -0,0 +1,25 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "host": "[%key:common::config_flow::data::host%]",
+          "port": "Port (usually 7000 or 8000)"
```

The label should be short:
```suggestion
          "port": "[%key:common::config_flow::data::host%]"
```
An explanation should be added in a `data_description` map instead

---

Done

---

You did not add a `data_description` though, you for example do like this:
```json
        "data": {
          "host": "[%key:common::config_flow::data::host%]",
          "port": "[%key:common::config_flow::data::port%]"
        }
        "data_description": {
          "port": "Usually 7000 or 8000"
        }
```

---

Added `data_description`